src/Components/dashboard/route/UpdateRouteButton.jsx

```diff
@@ -0,0 +1,247 @@
+                                </Typography>
+                                <TextField
+                                    id="outlined-basic"
+                                    variant="outlined"
+                                    placeholder= {route.data.data.routes.startLocation}
+                                    size="small"
+                                    sx={{ minHeight: 3, minWidth: 300 }}
+                                    onChange={(e) => setStartPoint(e.target.value)}
+                                    required
+                                    error={startPointError}
+                                />
+                            </Box>
+                            <Box sx={{ display: "flex", my: 2 }}>
+                                <Typography sx={{ width: 200,display:"flex", marginRight:"10px", justifyContent:"center", color: "blue", borderRadius: "50%", backgroundColor:"white", border: "2px solid blue", width:'25px', height:"25px" }}> B </Typography>
+                                <TextField
+                                    id="outlined-basic"
+                                    variant="outlined"
+                                    placeholder=   {route.data.data.routes.endLocation}
+                                    size="small"
+                                    sx={{ minHeight: 3, minWidth: 300 }}
+                                    onChange={(e) => setEndPoint(e.target.value)}
+                                    required
+                                    error={endPointError}
+                                />
+                            </Box>
+                            <Box sx={{ display: "flex" }} className="added-route">
+                                <Typography sx={{ width: 50 }}>Route Stops </Typography>
+                                <TextField
+                                    className="textfield-addroute"
+                                    id="outlined-basic"
+                                    variant="outlined"
+                                    placeholder="Enter Route Stop"
+                                    label=""
+                                    size="small"
+                                    sx={{ minHeight: 3, minWidth: 250 }}
+                                    onChange={(e) => setStop(e.target.value)}
+                                    required
+                                    error={stopError}
+                                />
+                                <AddIcon sx={{ py: 1, px: 1 }} onClick={handleAddRoute} />
+                            </Box>
+                            <Box>
+                                <List className="list-route">
+                                    <ListItem
+                                        sx={{ border: "1px solid #c4c4c4", width: "62%" }}
+                                    // secondaryAction={
+                                    //   // <IconButton edge="end" aria-label="delete">
```

Comment:
Let's remove these commented codes in this file, if they are not still needed

Reply:
Thank you for the comment @abraham-kwizera , I have resolved this